x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations.tsx

```diff
@@ -82,22 +101,46 @@ export const SuppliedConfigurations = () => {
     ];
   }, [modules]);
 
-  const schema = {
-    strict: true,
-    fields: {
-      tags: {
-        type: 'string',
+  const schema = useMemo(
+    () => ({
+      strict: true,
+      fields: {
+        tags: {
+          type: 'string',
+        },
       },
+    }),
+    []
+  );
```

Comment:
This looks like it's just initialized once without any options, could it be moved outside the component without the `useMemo()`?

Reply:
Oh good catch - updated in 643488707a8c517acfb689574f11f4024fc53c9b